sebastien15/capstoneproject
# Ft retrieve all queries

#### What does this PR do?

> This PR retrieve all queries from the database

#### Description of Task to be completed

> added a route to retrieve all queries
>add method to retrieve all queries

#### How should this be manually tested?

- Clone this repo with `git clone git@github.com:sebastien15/myportifolio.git` 
- Checkout to `ft-retrieve-all-queries`
- npm install nodemon -g
- npm install
- npm start
- run localhost:3000/contacts in post man make sure it is a get method

#### Any background context you want to provide?
>N/A
#### What are the relevant Trello story?
>https://trello.com/c/iPS9ZIxK

#### Screenshots (if appropriate)
>![get all contact](https://user-images.githubusercontent.com/41580903/94004004-4f934b00-fd9c-11ea-94e4-dd7efd5cc301.PNG)
